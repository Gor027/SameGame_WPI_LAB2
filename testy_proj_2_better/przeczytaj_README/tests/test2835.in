' '  'p'' 'pp''p   p '   p
p' 'ppp ppp'' pp 
 ppp' p   '   p' p
p 'p p'p  pp' p''  p'  ''p''p''pp pp'''pppp'p' '   'p   pp'''
p p'     ppp pp' pp  p 'p' p p p p'p''p' p'p'' '  ''ppp 
'    pp p'p ''p' 'p'p'p'  ' pp '
  p p'p
   pp'p'''
  p  p'''' '' ' ppp''p' pp''' ''''pp ''p p'pp p  ppp '  ''p'  pppp'  '  '' 'p p 
pp'' ' p 'p''''''p' pp'''p pp' p'' p  p'pp''p p p''p'p'ppp '' '  'p  pp p'''' 
