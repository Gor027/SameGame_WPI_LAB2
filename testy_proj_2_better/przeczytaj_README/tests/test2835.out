              '
     '    '   pp
     p   pppp pp
'    pp  p''' p' p   '   p     p
p '  pp' 'pp''p' ppp'p ''p 'p' p   p  'p pp'p      '
p p' 'p'''pp'pp'''ppp'p'p''ppp 'pppp 'p' p'p'     ''pp  pp '
p '' '''''p'''''ppp''p''pp'''p'''''p'''ppp'pp'p' pppp'p '''''p  p'' '' 'p
'''''''p''p''''''p''pp'''p'pp''p''pp''p'pp''p'p'p''p'p'pppp'''pp'pppp''''p
